docs/sections/contributor_guide/developer_setup.rst

```diff
@@ -6,7 +6,7 @@ Creating a ``bash`` Development Shell
 
 If an existing conda (:miniforge:`Miniforge<>`, :miniconda:`Miniconda<>`, :anaconda:`Anaconda<>`, etc.) installation is available and writable, step 1 may be skipped.
 
-.. include:: /shared/miniforge_instructions.rst
+#. .. include:: /shared/miniforge_instructions.rst
```

Comment:
The numbered list was not being rendered correctly when the `#.` was in the included file. Moving it from the included file to the including file fixes it.